queries.sql

```diff
@@ -72,4 +72,51 @@ GROUP BY neutered;
 select species, min(weight_kg) as minimum, max(weight_kg) as maximum from animals 
 GROUP BY species
 select species, avg(escape_attempts) from animals 
-where extract(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species
\ No newline at end of file
+where extract(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species
+
+/*Queries with JOINs*/
+-- What animals belong to Melody Pond?
+SELECT name, full_name 
+    FROM animals 
+    INNER JOIN owners 
+    ON animals.owner_id = owners.id 
+    WHERE full_name = 'Melody Pond';
+
```

Comment:
- [ ] As requested by the previous reviewer, please select all attributes instead one by one. As hint the command should be something like 👇
```sql
SELECT *
    FROM animals 
    INNER JOIN owners 
    ON animals.owner_id = owners.id 
    WHERE full_name = 'Melody Pond';
```